effectiveOfficeBackend/src/main/kotlin/office/effective/common/notifications/FcmNotificationSender.kt

```diff
@@ -29,6 +45,58 @@ class FcmNotificationSender(private val fcm: FirebaseMessaging): INotificationSe
             .build()
         fcm.send(msg)
     }
+    
+    override fun sendUpdateContentMessages(topic: String, resourceId: String) {
+        logger.info("[fcmNotificationSender] Received update on {} calendar id", resourceId)
```

Comment:
The name of the class is automatically specified by the log. The name of the method should be specified at the beginning of the message: [sendUpdateContentMessages]